augur/data/schema-multiplot.json

```diff
@@ -0,0 +1,126 @@
+                            "type": "string"
+                        },
+                        "threshold": {
+                            "description": "Default options for the measurement threshold. Optional -- if not provided then threshold will not be included and threshold controls will be hidden",
+                            "type": "object",
+                            "additionalProperties": false,
+                            "required": ["show", "value"],
+                            "properties": {
+                                "show": {
+                                    "description": "Should the threshold line be displayed by default?",
+                                    "type": "boolean"
+                                },
+                                "value": {
+                                    "description": "The numeric value of the measurements threshold",
+                                    "type": "number"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "collections": {
+            "description": "Collections of measurements of different types, conditions, adn/or experiments",
+            "type": "object",
+            "minProperties": 1,
+            "patternProperties": {
+                "^.*$": {
+                    "description": "A single collection of measurements",
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "description": "A single measurement for a sample and it's associated metadata. At least one of the metadata properties should be a group by option",
+                        "type": "object",
+                        "required": ["strain", "value"],
+                        "minProperties": 3,
+                        "properties": {
+                            "strain": {
+                                "description": "The name of the sample that matches the name of the sample within the tree",
+                                "type": "string"
+                            },
+                            "value": {
+                                "description": "The numeric value of the measurement",
+                                "type": "number"
+                            }
+                        },
+                        "patternProperties": {
+                            "^.*$": {
+                                "description": "Metadata associated with the measurement. Only metadata properties included in the groupings config will be included in the group by dropdown, but all metadata properties will be available as a filter",
+                                "type": ["string", "number", "boolean"]
```

Comment:
I hadn't considered non-string values until now, but do you want to support this? Will there be any difference in behavior between `"42"` and `42`?

Reply:
The separate data types would allow the downloaded data TSV to keep the user intended data types. If we don't want to support this, I think this can definitely just all be "string".